EIPS/eip-5450.md

```diff
@@ -21,59 +21,102 @@ Currently existing EVM implementations perform a number of validity checks for e
 
 In particular, this extended code validation eliminates the need for EVM stack underflow checks done for every executed instruction. It also prevents deploying code that can be statically proven to require more than 1024 stack items, however it may still be possible to exceed that limit in some cases, and thus overflow checks cannot be fully eliminated.
 
+
 ## Specification
 
 ### Code validation
 
-*Remark:* We rely on the notions of *data stack* and *type section* as defined by [EIP-4750](./eip-4750.md).
+*Remark:* We rely on the notions of *operand stack* (FIXME) and *type section* as defined by [EIP-4750](./eip-4750.md).
+
+Each code section is validated independently.
+
+#### Instructions validation
```

Comment:
Not sure why we have to repeat this validation in this EIP, it is alreadt specified in others.

Reply:
I wanted a holistic overview and show that 2 passes are needed.

In EIP-4200 it is specified what is valid but not how to validate. 

> verify that each RJUMP/RJUMPI has a relative_offset pointing to an instruction. This means it cannot point to an immediate data of PUSHn/RJUMP/RJUMPI. It cannot point outside of code bounds.

It suggests 2 passes. Here I show how to do it with single pass. This is to visualize the complexity of the algorithm.

Reply:
I would say it's good for a separate "Implementation recommendations" document, but maybe not for the spec. It's confusing to repeat the same things in several EIPs, they look conflicting.

I don't think EIP-4200 suggests any specific algorithm.

Reply:
"Unified spec" document can be a good place for this.

Reply:
I agree. Here you can import something from eip3670.

Reply:
This is now removed.